py/selenium/webdriver/chromium/virtual_authenticator.py

```diff
@@ -0,0 +1,248 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from base64 import urlsafe_b64encode, urlsafe_b64decode
+from enum import Enum
+
+
+class Protocol(Enum):
+    """
+    Protocol to communicate with the authenticator.
+    """
+    CTAP2 = "ctap2"
+    U2F = "ctap1/u2f"
+
+
+class Transport(Enum):
+    """
+    Transport method to communicate with the authenticator.
+    """
+    BLE = "ble"
+    USB = "usb"
+    NFC = "nfc"
+    INTERNAL = "internal"
+
+
+class VirtualAuthenticatorOptions:
+
+    Protocol = Protocol
```

Comment:
No they are not Necessarily have to go here. But my thinking of adding this here has to do with using the Protocol and Transport stuff

Suppose we are not adding it as part of class then I have to import them in Chrome, Edge too and supply the Enum stuff from there. 

So example would be
```python
from selenium.webdriver.chrome.webdriver import WebDriver
from selenium.webdriver.chrome.virtual_authenticator import (
VirtualAuthenticatorOptions, Credential, Transport, Protocol )

options = VirtualAuthenticatorOptions()
options.protocol = Transport.BLE
driver = WebDriver()
driver.add_virtual_authenticator(options)
```
But now it will be more similar to Java like 
```python
from selenium.webdriver.chrome.webdriver import WebDriver
from selenium.webdriver.chrome.virtual_authenticator import (
VirtualAuthenticatorOptions, Credential)

options = VirtualAuthenticatorOptions()
options.protocol = VirtualAuthenticatorOptions.Transport.BLE
driver = WebDriver()
driver.add_virtual_authenticator(options)
```
@AutomatedTester what are your views on this

Reply:
I think we should have this like Java here. It helps guide people through how to use it.